apps/vscode/package.json

```diff
@@ -1382,7 +1392,7 @@
     "@types/semver": "^7.3.13",
     "@types/tmp": "^0.2.3",
     "@types/uuid": "^9.0.0",
-    "@types/vscode": "1.66.0",
+    "@types/vscode": "1.94.0",
```

Comment:
Since it is in dev dependencies, I was not under the impression it will force people into a new vscode version? 

For CI purposes I can back down to a lower version, like 1.75.0, but to differentiate between a diff ediotr we do need the `vscode.window.tabGroups` API

Reply:
Oh, then also note that this is in that file

```
  "engines": {
    "vscode": "^1.75.0"
  },
```

stating that Quarto has compatibility with vscode 1.75.0 and up. But if we unconditionally use types from a newer vs code, then that doesn't hold true

I'm not sure what happens if we try `window.tabGroups.activeTabGroup.activeTab?.label` when `window.tabGroups` doesn't exist